### DISCUSSION TITLE
Issue when pausing during gameplay

### DISCUSSION THREAD

**Discussion by 770grappenmaker** [ppy/osu#21038] [Q&A] [answered]
When pausing the game during gameplay, you obviously have to click the location of your cursor before the pausing happened. But actually, when you pause while your cursor is above a circle/slider, if you unpause the click of the unpause counts. In the case that the circle that gets "ghost clicked" was already visible but the click is way too early, this counts as a miss. Not sure if this "ghost click" mechanic is a bug/oversight, or intentional.

Edit: apparently, it does count as a miss, however, the slider/circle does not disappear. You can click the circle/slider again, and it'll actually give you a judgement... interesting.

**Comment by Joehuu** [comment, marked as answer]
This is already tracked in https://github.com/ppy/osu/issues/9658.

**Reply by 770grappenmaker** [reply]
Whoops, my bad!
